include/clap/events.h

```diff
@@ -297,11 +297,25 @@ typedef struct clap_event_midi {
    uint8_t  data[3];
 } clap_event_midi_t;
 
+// clap_event_midi_sysex contains a pointer to a sysex contents buffer.
+// The lifetime of this buffer is (from host->plugin) only the process
+// call in which the event is delivered or (from plugin->host) only the
+// duration of a try_push call.
+//
+// Since `clap_output_events.try_push` requires you to make a copy of
+// an event, host implementers receiving sysex messages from plugin need
+// to take care to both copy the event (so header, size, etc...) but
+// also allocate and memcpy the sysex pointer. Similarly plugins retaining
+// the sysex outside the lifetime of a single process call must copy the
```

Comment:
This stuff runs in the audio thread, so you're not supposed to use the memory manager. "allocate" does suggest this I think?
If a receiver is only interested in small sysex messages this can easily be solved, but a general purpose receiver should probably pre-allocate some memory.

Reply:
Yeah also copy to host owned memory is better. I’ll fix it up